core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/components/dot-experiments-configuration-traffic-split-add/dot-experiments-configuration-traffic-split-add.component.ts

```diff
@@ -127,8 +128,8 @@ export class DotExperimentsConfigurationTrafficSplitAddComponent implements OnIn
      */
     checkControl(arrayIndex: number, value: number): void {
         (
-            (this.form.get('variants') as FormArray).controls[arrayIndex] as FormGroup
-        ).controls.weight.setValue(value);
+          (this.form.get("variants") as FormArray).controls[arrayIndex] as FormGroup
+        ).controls["weight"].setValue(value);
```

Comment:
isn't `single quotes` the standard?

Reply:
Let me check ESLint, the IDE did that. Fixed.